entrypoint.sh

```diff
@@ -1,22 +1,41 @@
-#!/bin/sh -l
+#!/bin/bash
 
-echo ""
-echo "Installing dependencies specified in cpanfile"
-echo ""
+set -eo pipefail
```

Comment:
Awesome.

I even have a TIL/note on this, REF: http://jonasbn.github.io/til/bash/write_safe_shell_scripts.html

Reply:
It is not as good as `strict` and `warnings` pragmas in Perl, but it helps.